RELEASE_NOTES.md

```diff
@@ -2,6 +2,128 @@
 
 <!-- newest release at the top please) -->
 
+# Prod 2019/05/08
+## Ingest
+### Ingest-ui v0.9.2
+* Setting of row height automatically to show all validation errors
+* Do not allow creation of project metadata thru UI
+ 
+## Upload  v4.3.1
+* Remove "list upload area" REST API endpoint
+* Added client-side checksumming (temporarily disabled).
+* Much Internal refactoring and cleanup.
+
+## Data Store
+### Version: 2019-05-08-02-45-24-staging.release
```

Comment:
Version should be `2019-05-08-15-27-25-prod.release`

Reply:
Thanks Brian! I changed it.